src/brogue/Globals.c

```diff
@@ -2406,6 +2406,43 @@ const char armorRunicNames[NUMBER_ARMOR_ENCHANT_KINDS][30] = {
     "immolation",
 };
 
+// To meter item generation (on level generation):
+// incrementFreq must be != 0 for frequency biasing
+// lvlScale != 0 for thresholding
+//  { category, kind,                   initFreq,   incFreq,decFreq,gMulti, gInc,   lvlScale }
+const meteredItemGenerationTable meteredItemsGenerationTable[NUMBER_METERED_ITEMS] = {
```

Comment:
Since this is a brand new table, I think it could make sense to make use of designated initializers here:

```c
const meteredItemGenerationTable meteredItemsGenerationTable[NUMBER_METERED_ITEMS] = {
  { .category = SCROLL, .kind = SCROLL_ENCHANTING, .initialFrequency = 60, .incrementFrequency = 30, .decrementFrequency = 50 },
  { .category = SCROLL, .kind = SCROLL_IDENTIFY },
  { .category = SCROLL, .kind = SCROLL_TELEPORT },
  { .category = SCROLL, .kind = SCROLL_REMOVE_CURSE },
  // ...
  { .category = POTION, .kind = POTION_LIFE, .initialFrequency = 0, .incrementFrequency = 34, .decrementFrequency = 150, .genMultiplier = 4, .genIncrement = 3, .levelScaling = 1},
  // ...
};
```

Although it makes the table a little bit more verbose, it means that the comment explaining the struct field order is not needed, and it's now guaranteed to be kept in-sync. Also, any zero fields (which is most of them) can be omitted entirely.

For easy-of-scanning, aligning them with spaces still makes sense, it will just be wider than it is now.

Reply:
I like this change, view @tmewett ?

Reply:
I don't mind!